packages/wallet/src/pages/account/[id].tsx

```diff
@@ -0,0 +1,141 @@
+import { PageHeader } from '@/components/PageHeader'
+import { PaymentPointerCard } from '@/components/PaymentPointerCard'
+import { useDialog } from '@/lib/hooks/useDialog'
+import { Link } from '@/ui/Link'
+import type {
+  GetServerSideProps,
+  InferGetServerSidePropsType
+} from 'next/types'
+import { z } from 'zod'
+
+type AccountPageProps = InferGetServerSidePropsType<typeof getServerSideProps>
+
+export default function AccountPage({ account }: AccountPageProps) {
+  const [openDialog, closeDialog] = useDialog()
+  return (
+    <AppLayout>
+      <div className="flex items-center justify-between md:flex-col md:items-start md:justify-start">
+        <PageHeader
+          title="Hello, John Doe!"
+          message="Here is your account overview!"
+        />
+        <div className="text-green md:mt-10">
+          <h2 className="text-lg font-light md:text-xl">Balance</h2>
+          <p className="text-2xl font-semibold md:text-4xl">
+            {account.balance}
+          </p>
+        </div>
+      </div>
+      <div className="flex w-full flex-col space-y-3 md:max-w-md">
+        <div className="mt-2 flex justify-between space-x-2">
+          <button
+            onClick={() =>
+              openDialog(<CreatePaymentPointerDialog onClose={closeDialog} />)
+            }
+            className="group flex aspect-square h-24 w-24 flex-col items-center justify-center -space-y-1 rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6"
+          >
+            <New className="h-7 w-7" />
+            <div className="-space-y-2">
+              <p className="font-medium text-green-5 group-hover:text-green-6">
+                Add payment{' '}
+              </p>
+              <p className="font-medium text-green-5 group-hover:text-green-6">
+                pointer
+              </p>
+            </div>
+          </button>
+          <Link className="group flex aspect-square h-24 w-24 flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Receive className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Fund
+            </span>
+          </Link>
+          <Link className="group flex aspect-square h-24 w-24 flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Exchange className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Exchange
+            </span>
+          </Link>
+        </div>
+        <div className="flex items-center justify-between">
+          <h3 className="text-lg font-semibold leading-none text-green">
+            Account
+          </h3>
+        </div>
+        <div className="flex items-center justify-between rounded-md bg-gradient-to-r from-[#92DBCA] to-[#56B1AF] px-3 py-2">
```

Comment:
couldn't we use a gradient we already have in the config?

Reply:
Totally missed that. Good catch.